supporting_content/columns/servicesubcategory.md

```diff
@@ -0,0 +1,189 @@
+|                           | Business Intelligence             | QuickSight                                                                | Power BI, Analysis Services                                                                                                                                                             | Looker                                                             |
+|                           | Data Processing                   | EMR, Batch, Glue, Data Pipeline                                           | Databricks, HDInsight, Data Lake Analytics                                                                                                                                              | Cloud Composer, Dataflow, Dataproc                                 |
+|                           | Search                            | Cognitive Search, CloudSearch, Elasticsearch, Kendra                      | Azure AI Search                                                                                                                                                                         | Vertex AI Search                                                   |
+|                           | Streaming Analytics               | Kinesis                                                                   | Stream Analytics, Data Explorer                                                                                                                                                         |                                                                    |
+|                           | Other (Analytics)                 | Clean Rooms                                                               | Genomics, Data Lake Analytics                                                                                                                                                           |                                                                    |
+| Business Applications     | Other (Business Applications)     | WorkDocs, WorkLink, WorkMail, Chime                                       | Microsoft 365, Teams                                                                                                                                                                    | Workspace                                                          |
+| Compute                   | Containers                        | EKS, ECS                                                                  | AKS, Container Apps, Container Instances, Container Registry, Container Service, Service Fabric                                                                                         | GKE, Cloud Run                                                     |
```

Comment:
OKE?

Reply:
I will admit that I did not go through as exhaustive an exploration of OCI services as I did AWS, GCP, and Azure -- but I did spot check, and we did include OCI in many of the task force discussions.

If you wanted to add a column to [this spreadsheet](https://docs.google.com/spreadsheets/d/1aYq79sWp8TK4zbcSKUb-gRIBei3fERVwAgDJ5ezZf1w/edit?usp=sharing) and take a first stab at where you think OCI services will belong in this taxonomy, then I'd be happy to partner with you on that in getting a PR stood up!  But I think at this point, it would be a separate PR that we would want to tackle outside of this one.  So, I'm resolving this comment, pending your interest in going down that road!